### ISSUE TITLE
[COGNITION]: Add aria-describedby and SR-only instructions to the calendar widget for better semantic meaning

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6100]
## Description

I'd like to add some semantic HTML and adjust the ARIA attributes on our calendar widgets. We are getting good results with NVDA, VoiceOver and iOS VoiceOver, but JAWS does not honor the arrow keys and point to point navigation with virtual cursor. The `TAB` navigation paradigm works well, and form inputs show up in the right menus. Adding some user instructions for screen readers will be a big benefit. Code snippet and screen shot attached below.

/cc @peggygannon because the helper text I'm proposing will need a look over for plain language.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/2709
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/6102

## Point of Contact

**VFS Point of Contact:** _Trevor, Peggy_

## Acceptance Criteria

- [x] Screen readers read out the month and year of the table, then pause, then read out the instructions for use.
- [x] The visual display and functionality are not disrupted by the code adjustments
- [ ] The calendar widget (and page) does not throw any axe scan errors

## Environment

* Windows 10
* IE11, Chrome
* JAWS
* calendar widgets (direct booking, and AM/PM checkbox) views

## Possible Fixes (optional)

Adjusting the table container and adding a paragraph for SR-only caption text should work great:

<div aria-describedby="h2-2020-02" class="vaos-calendar__container vads-u-margin-bottom--3" role=
"table">
  <h2 class="vads-u-font-size--h3" id="h2-2020-02">February 2020</h2>
  
  
  
  <div class="usa-sr-only" id="caption-1">
    Press the Enter key to expand the day you want to schedule an appointment. Then press the Tab
    key or form shortcut key to select an appointment time.
  </div>
  ...
  <div role="rowgroup">
    <div>
      <div class="vads-u-flex-wrap--wrap vads-u-display--flex..." role="row">
        <div class="vaos-calendar__calendar-day" role="cell" style="height: auto;">
          
        
        
          <button
            aria-expanded="false"
            aria-describedby="caption-1"
            aria-label="Monday, February 3rd"
            class="vaos-calendar__calendar-day-button"
            disabled
            id="date-cell-2020-02-03"
            type="button"
          >
            3
          </button>
        </div>
        ...
      </div>
    </div>
  </div>
</div>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs

<!-- Drop any screenshots or error logs that might be useful for debugging -->
![Screen Shot 2020-02-26 at 4 45 54 PM](https://user-images.githubusercontent.com/934879/75395429-06d3ec00-58b8-11ea-9853-7066721ebd73.png)

**Comment by 1Copenut** [comment]
@lenaecb && @jbalboni I retested this with JAWS (IE11, Chrome) and got part of the way through with NVDA (Chrome) before getting some environment errors. It should be good to go, but I'd like to get a quick confirmation with NVDA+Firefox before closing.
